spk/flexget/Makefile

```diff
@@ -1,16 +1,18 @@
 SPK_NAME = flexget
-SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 5
+SPK_VERS = 2.9.14
+SPK_REV = 6
 SPK_ICON = src/flexget.png
 
+# Python wheels
+# https://github.com/SynoCommunity/spksrc/wiki/Using-wheels-to-distribute-Python-packages
```

Comment:
No, documentation is never necessary. As I would have been happy to find such lines there at first contact to help me spare time, I wrote them thinking at future next maintainer...